experimental/GroebnerWalk/results.csv

```diff
@@ -0,0 +1,30 @@
+name,standard_walk,generic_walk,perturbed_walk,buchberger
+simple,0.002634119,0.001080056,,4.670857142857143e-6
+simple,0.00262981,0.001087529,,4.621e-6
+simple,0.000623,0.000195084,,3.0837068965517243e-7
+cyclic5-QQ,0.01911975,0.110495292,,8.884516129032258e-7
+cyclic5-Fp,0.01191775,0.251762958,,8.766274509803922e-7
+simple,0.000620541,0.000199625,,3.2019369369369374e-7
+cyclic5-QQ,0.019888625,0.116872417,,8.674242424242424e-7
+cyclic5-Fp,0.01215325,0.268002959,,8.661276595744681e-7
+cyclic7-Fp,36.670678,,,
+simple,0.000641,0.000202875,,3.0515983606557375e-7
+cyclic5-QQ,0.020051542,0.117901583,,8.731739130434783e-7
+cyclic5-Fp,0.013118917,0.261012125,,8.7975e-7
+cyclic6-QQ,0.356129125,5.922552333,,1.1916e-6
+cyclic6-Fp,0.130183917,11.620150625,,1.15e-6
+simple,0.000753417,0.000253458,,4.2939408866995077e-7
+simple,0.000753125,0.000237833,,4.0463111111111113e-7
+simple,1.183723958,0.269548625,,2.2375e-5
+agk4-QQ,10.771205958,55.705554,,527.126850292
+agk4-Fp,3.302066417,49.991833833,,0.095817208
+katsura6-QQ,1.003294291,,35.098429459,
+simple,1.422464625,0.274453459,,2.475e-5
+agk4-QQ,11.076574625,55.562715875,,543.319831
+agk4-Fp,3.492181375,51.427781875,,0.094963834
+tran3.3-QQ,0.640738125,24.688099959,,2323.13229425
+tran3.3-Fp,0.254231917,14.402583584,,0.003486333
+newellp1-QQ,51.5992475,1529.494271334,,811.051479458
+newellp1-Fp,9.365612875,1140.743606416,,0.165012125
+simple,1.149968916,0.25867925,,2.5334e-5
+katsura6-QQ,0.813826709,35.404766583,,
```

Comment:
I don't really see the benefit of having this file in Oscar. It seems more like this is your personal benchmark result that's used in some paper.

Reply:
While this is in experimental, I would keep that file. It can certainly help to detect regression, because it incorporates information on standardized examples.

The question, on how to keep or drop this data later on, when this package moves to src, is not relevant for the moment.